docs/en/setup/backend/backend-alarm.md

```diff
@@ -47,6 +50,17 @@ For example, in **percentile**, `value1` is the threshold of P50, and `-, -, val
 - **Silence period**. After the alarm is triggered at Time-N (TN), there will be silence during the **TN -> TN + period**.
 By default, it works in the same manner as **period**. The same Alarm (having the same ID in the same metrics name) may only be triggered once within a period. 
 
+Such as for a metric, there is a shifting window as following at T7.
+
+| T1     | T2     | T3     | T4     | T5     | T6     | T7     |
+|--------|--------|--------|--------|--------|--------|--------|
+| Value1 | Value2 | Value3 | Value4 | Value5 | Value6 | Value7 |
+
+* `Period`(Time point T1 ~ T7) are continuous data points for minutes. Notice, alerting doesn't support for hours/days metrics as pointless.
```

Comment:
The wording here is a bit odd. Specifically: `Notice, alerting doesn't support for hours/days metrics as pointless.`.

Maybe something like:

```suggestion
* `Period`(Time point T1 ~ T7) are continuous data points for minutes. Notice, alerts are not supported above minute-by-minute periods as these would not be efficient.
```